src/util/member.ts

```diff
@@ -0,0 +1,5 @@
+import type { GuildMember } from 'discord.js';
+
+export const memberHasRoles = (member: GuildMember, ...roles: string[]): boolean => {
```

Comment:
just make roles an actual array and not spread parameters. i don't see the benefit here. then when it's called it's just
```ts
    const hasRequiredRoles = memberHasRoles(newMember, [roleA, roleB]);
```
or 
```ts
    const hasRequiredRoles = memberHasRoles(newMember, [someRole]);
```